src/components/SideMenu.tsx

```diff
@@ -25,7 +25,7 @@ export const MinimizedMenuIconButton = ({ children, ...props }: IconButtonProps)
 };
 
 export const SideMenu = ({ children, expanded, minimizedMenu }: SideMenuProps) => (
-  <Box component="section" sx={{ minWidth: '5rem' }}>
+  <Box component="nav" aria-labelledby="side-menu-header" sx={{ minWidth: '5rem' }}>
```

Comment:
Gikk for å både wrappe hele menyen i en `nav` og ha headers. Siden menyen ikke kun inneholder navigasjon tenker jeg det gir mening å beskrive innholdet som finnes under tittelen.